code/modules/reagents/chems/chems_painkillers.dm

```diff
@@ -95,14 +112,14 @@
 
 /decl/material/liquid/painkillers/affect_overdose(var/mob/living/M)
 	..()
-	if(!narcotic)
-		return
-	M.set_hallucination(120, 30)
 	SET_STATUS_MAX(M, STAT_DRUGGY, 10)
 	M.add_chemical_effect(CE_PAINKILLER, pain_power*0.5) //extra painkilling for extra trouble
-	M.add_chemical_effect(CE_BREATHLOSS, breathloss_severity*2) //ODing on opiates can be deadly.
-	if(isboozed(M))
-		M.add_chemical_effect(CE_BREATHLOSS, breathloss_severity*4) //Don't drink and OD on opiates folks
+	M.add_chemical_effect(CE_TOXIN, 1)
```

Comment:
I don't *think* ODing on opiates is toxic? The breathloss is pretty punishing already, isn't it? I'd either put this behind an `else` after `if(narcotic)` or give it its own separate overdose toxicity var.